pkg/apps/application.jsx

```diff
@@ -141,16 +141,14 @@ export const Application = ({ metainfo_db, id, progress, progress_title, action
     }
 
     return (
-        <Page groupProps={{ sticky: 'top' }}
-              id="app-page"
-              className="application-details"
-              isBreadcrumbGrouped
-              breadcrumb={
-                  <Breadcrumb>
-                      <BreadcrumbItem to="#/">{_("Applications")}</BreadcrumbItem>
-                      <BreadcrumbItem isActive>{comp ? comp.name : id}</BreadcrumbItem>
-                  </Breadcrumb>
-              }>
+        <Page id="app-page"
+              className="application-details">
+            <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
+                <Breadcrumb>
+                    <BreadcrumbItem to="#/">{_("Applications")}</BreadcrumbItem>
+                    <BreadcrumbItem isActive>{comp ? comp.name : id}</BreadcrumbItem>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18683-20230421-074351-5ee2603c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)